compiler/src/main/resources/non-compiling-programs/multi-state-index.cell

```diff
@@ -0,0 +1,13 @@
+world {
+    size = 100 [wrap], 200 [wrap]
+    tickrate = 1
+    cellsize = 5
+}
+
+state ident 5 (0, 0, 0) {
+    if (# == 2) {
+        become ident;
```

Comment:
Shouldn't any use of ident have a lookup that specifies which one of the multistates to transition to?

Reply:
I suppose. The point is moot though, as the multi-state-declaration `5` has been deleted and the example now compiles and has been moved to compiling programs.

The formal semantics specify `#` to return the given index of a state. In the case of a single-state/non-multi-state, this would simply be 0.